website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -48,33 +48,28 @@ pip is the easiest way to install the adapter:
-|   `user`   | Starburst Enterprise & Trino<br></br>`user.name`<br></br>-OR-<br></br>`user.name@mydomain.com`<br></br>Starburst Galaxy<br></br>`user.name@mydomain.com/<role>` | The username to log into your Starburst Enterprise, Starburst Galaxy or Trino cluster.<br></br>The user must have permissions to create and drop tables.<br></br>When connecting to Starburst Galaxy clusters, the role of the user must be provided as a suffix to the username.<br></br>**NOTE**: When connecting to a Starburst Enterprise cluster with built-in access controls enabled, you will not be able to provide the role as a suffix to the username, so the default role for the provided username will be used. |
+|   `host`   | `mycluster.mydomain.com`                                                                                                                                        | The hostname of your cluster.<br/><br/>Don't include the `http://` or `https://` prefix.                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `database` | `my_postgres_catalog`                                                                                                                                           | The name of a catalog in your cluster.<br/><br/>The provided username must have read/write access to this catalog.<br/><br/>The selection you make doesn't limit the data you can access through dbt to the specified catalog in Starburst/Trino. By default, dbt models will be created and/or materialized within this catalog.                                                                                                                                                                                               |
+|  `schema`  | `my_schema`                                                                                                                                                     | The name of a schema within your cluster's catalog.<br/><br/>The provided username must have read/write access to this schema.<br/><br/>The selection you make doesn't limit the data you can access through dbt to the specified schema in Starburst/Trino. By default, dbt models will be created and/or materialized within this catalog.<br/><br/>It's _not recommended_ to use schema names that have upper case or mixed case letters.                                                                                                    |
```

Comment:
Because the below is mentioned for both `catalog` and `schema`, do you think it'd make sense to pull it out and mention it before the table?

> The selection you make doesn't limit the data you can access through dbt to the specified schema in Starburst/Trino. By default, dbt models will be created and/or materialized within this catalog.

Reply:
I'm also now interested to know if this content aligns with the "Connect to Starburst" page we made with a similar table

Reply:
good call! made snippets for the pulled out sections; the section headings were at diff levels (H3 vs H2) so accommodated that too.